helpers/modalHelper.js

```diff
@@ -0,0 +1,11 @@
+const { setReminder } = require('./reminderHelper')
+
+async function replyToModal (interaction) {
+  if (interaction.fields.getTextInputValue('reminderTime')) {
```

Comment:
I think it would be cleaner to check this with:`interaction.customId==='reminderModal'`

This can then in turned be improved even further by adding an Ids object to the const file so you can reference it like
`interaction.customId === constants.ids.modals.reminder`

Reply:
We'd have to rename the customId to reminderModal then, which would look weird when building up the modal in the first place. Open to discuss this though.

Reply:
On a second look this does indeed look way cleaner and the text-boxes don't have to be renamed as I incorrectly assumed. Fixed!